Spectrum-Analyzer/Spectrum-Analyzer/Src/Dsp/SpectrumTypes/PhaseSpectrum.cpp

```diff
@@ -0,0 +1,45 @@
+#include "PhaseSpectrum.hpp"
+// #include <math.h> (Uncomment when implemented)
+
+PhaseSpectrum::PhaseSpectrum(int iBufferSize_, int iSampleRate_) 
+   : SpectrumBase(iBufferSize_, iSampleRate_)
+{
+   
+   return; 
+}
+
+void PhaseSpectrum::CalculateSpectrum()
+{
+   //const float fPhaseThreshold = 0.0f;
+
+   //!TODO: No idea what this code is even doing at the moment. 
```

Comment:
Interesting we should look into this....

Reply:
!!
Calculating phase hard, code was not doing anything that made sense. Moved and commented out for now.